### PR TITLE
hometask1

### PR DESCRIPTION
**Pull request by vladshavel**
how to delete .DS_Store?

### CODE REVIEW

**Comment by vladshavel** [comment]
and how i can delete extra commits?

**Comment by pluhin** [comment]
> how to delete .DS_Store?

Add into .gitignore file, one line one pattern
Example for file
```
 .DS_Store
```

**Comment by pluhin** [comment]
> and how i can delete extra commits?

You can squash them (like merging) : here is example  https://www.devroom.io/2011/07/05/git-squash-your-latests-commits-into-one/

File: vshavel/addpart.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+‐‐‐‐‐‐‐‐‐‐‐
+   | х |
+‐‐‐‐‐‐‐‐‐‐‐
+ o | x | o</pre>
+
+ <hr>
+
+<h3>Тег address</h3>
+
+<p><i>Kuprevicha, 1/1, 220141, Minsk, Belarus. Phone: +375.17.2686911</i></p>
```
Comment (pluhin) [inline vshavel/addpart.html]: Here need tag address

File: vshavel/addpart.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+<hr>
+
+<h3>Семантические и функциональные строчные теги</h3>
+
+<p>Пожелания автора:
+<br><s style="color:#FF0000">Данный текст убрать из монографии</s> и заменить его на <u style="color:#0000CD">вновь помещённый текст</u>.</p>
```
Comment (pluhin) [inline vshavel/addpart.html]: Need semantic tags here del and ins

File: vshavel/addpart.html [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+                занимающий более одной строки ‐‐&#62;</pre>
+
+                <hr>
+
+
+<h3>Теги над­ - и подстрочных символов</h3>
+
+<div align="center">
+
+<p><b>Магазин работает:</b></p>
```
Comment (pluhin) [inline vshavel/addpart.html]: Header here

**Review by pluhin — CHANGES_REQUESTED** [review]

File: vshavel/addpart.html (hunk 1 above)
Comment (vladshavel) [reply vshavel/addpart.html]: should i sent somewhere changed files?

File: vshavel/addpart.html [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+<h3>Семантические и функциональные строчные теги</h3>
+
+<p>Пожелания автора:
+<br><del style="color:#FF0000">Данный текст убрать из монографии</del> и заменить его на <ins style="color:#0000CD">вновь помещённый текст</ins>.</p>
+
+<hr>
+
+<h3>Работа со строчными и блоковыми цитатами</h3>
+<p>Цитата короткая:<br>
+<i>И всё­таки она вертится!</i></p>
```
Comment (pluhin) [inline vshavel/addpart.html]: tag q or tag quote here

File: vshavel/addpart.html [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+	    Готовят просвещенья дух<br>
+	    И опыт, сын ошибок трудных,<br>
+	    И гений, парадоксов друг,<br>
+	    И случай, бог изобретатель...</p></blockquote>
+
+      <hr>
+
+<h3>Семантические и функциональные строчные теги</h3>
+
+<strong>Тэг &#60;STRONG&#62; делает данный фрагмент текста более важным.</strong> <em>Тэг &#60;em&#62; менее важным.</em>
```
Comment (pluhin) [inline vshavel/addpart.html]: add paragraph here

File: vshavel/addpart.html [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+<hr>
+
+
+<h3>Акроним и курсив</h3>
+
+<p>ТФ: Y(x)=e<sup><i>-iπx</i></sup>+log<sub><i>x</i></sub>4</p>
+
+<hr>
+
+<h3>Тег small внтури абзаца</h3>
```
Comment (pluhin) [inline vshavel/addpart.html:120]: "Тег small внтури абзаца" - it mean need to use tag ```small``` inside  tag ```p```, not only ```h3```

File: vshavel/part1.html [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+print 'Hello. I am a python program.'
+name = raw_input("What is your name?")
+print "Hello there, " + name + "!"</pre>
+
+<p>Это всё. Сохраните файл как <strong>hello.py</strong> в любом месте. Я предлагаю сохранить его в папке
+<strong>python_examples</strong> в домашнем каталоге. Этот пример показывает, как просто написать
+программу на Python. Для запуска программы нужно сделать её файл исполняемым. Для
+этого введите в терминале</p>
+
+<u><i>chmod +x hello.py</i></u>
```
Comment (pluhin) [inline vshavel/part1.html]: tag pre here

File: vshavel/part1.html [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+    print cntr</pre>
+
+<p><i>Не забудьте поставить символ табуляции перед строкой «print cntr».</i> Это важно. В
+Python для указания на блоки кода используются не круглые «(» и фигурные «{» скобки
+как в других языках программирования, а отступы.</p>
+
+
+</body>
+</html>
+
```
Comment (pluhin) [inline vshavel/part1.html:91]: extra line here

**Review by pluhin — CHANGES_REQUESTED** [review]

**Review by pluhin — CHANGES_REQUESTED** [review]
Good, but you didn't checks my comments

**Comment by vladshavel** [comment]
i corrected this mistakes that you indicate in comments

**Comment by pluhin** [comment]
> i corrected this mistakes that you indicate in comments

Sorry, I see

**Review by pluhin — APPROVED** [review]

**Pull request merged**